frameworks/Python/litestar/benchmark_config.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
Maximum 10 variants.
#8420

Reply:
I have changed the `benchmark_config.json` @joanhey